addons/mail/static/src/core/common/thread_model.js

```diff
@@ -281,6 +282,18 @@ export class Thread extends Record {
     canPostOnReadonly;
     /** @type {luxon.DateTime} */
     last_interest_dt = Record.attr(undefined, { type: "datetime" });
+    /** @type {luxon.DateTime} */
+    lastInterestDt = Record.attr(undefined, {
+        type: "datetime",
+        compute() {
+            const selfMemberLastInterestDt = this.selfMember?.last_interest_dt;
+            const lastInterestDt = this.last_interest_dt;
+            return compareDatetime(selfMemberLastInterestDt, lastInterestDt) > 0
+                ? selfMemberLastInterestDt
+                : lastInterestDt;
+        },
+    });
+    // Something else here
```

Comment:
what is that comment?

Reply:
This is a draft PR and not finished yet.... hahaha :sweat_smile: I didnt expect you review that. I will try to have it today or tomorrow so that it can be done before the freeze